MIAUI/MIAUI/TasksRepository.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace MIAUI;
+
+public class TasksRepository
```

Comment:
I recommend moving this file into a new folder called "Data"

See https://github.com/xamarin/xamarin-forms-samples/tree/main/Todo/Todo/Todo as an example